lib/readline.js

```diff
@@ -390,7 +390,10 @@ Interface.prototype._tabComplete = function() {
         var width = completions.reduce(function(a, b) {
           return a.length > b.length ? a : b;
         }).length + 2;  // 2 space padding
-        var maxColumns = Math.floor(self.columns / width) || 1;
+        var maxColumns = Math.floor(self.columns / width);
+        if (!maxColumns || maxColumns === Infinity) {
+          maxColumns = 1;
+        }
```

Comment:
While I see that this fixes the issue, I'm not sure exactly why it does so. With `self.columns` being undefined, the division should result in `NaN`. Care to explain why it results in `Infinity`?